src/pages/namsunkim/MainRight/MainRight.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { defaultUser, prefix } from '../config';
+import RecommendAccount from './Components/RecommendAccount';
+import './MainRight.scss';
+
+export default class MainRight extends Component {
+
+  constructor() {
+    super();
+    this.state = {
+      recommendAccount: [],
+    }
+  }
+
+  componentDidMount() {
+    const { api } = prefix;
+    fetch(`${api}/recommendAccountData.json`)
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          recommendAccount: data,
+        });
+      });
+  }
+
+  render() {
+    const { recommendAccount } = this.state;
+    const { userName } = this.props;
+    const { profile } = prefix;
+
+    return (
+      <>
+        <aside className="mainRight">
+          <div className="account">
+            <img src={profile + defaultUser.profile}
+              alt={`${userName}님의 profile`} />
+            <div className="idWrap bold">
+              <span>{userName ? userName : defaultUser.userName}</span>
```

Comment:
```suggestion
              <span>{userName || defaultUser.userName}</span>
```
삼항연산자로 표현할 필요 없이 OR 연산자로 처리 가능한 구문입니다~!

Reply:
수정하겠습니다!